rust-create-cascade/src/main.rs

```diff
@@ -159,6 +159,8 @@ fn include<T>(
             builder
                 .include(key)
                 .expect("Capacity error. Did the file contents change?");
+            // Ensure that we do not attempt to include this issuer+serial again.
+            revoked_serial_set.remove(serial);
```

Comment:
Would we ever have the same issue in `exclude`?

Reply:
Good question. We will not, because we count entries in the "known" list with multiplicity https://github.com/mozilla/crlite/blob/cee502d2c47e0f2bde17cc71d3f7db80fc87777f/rust-create-cascade/src/main.rs#L105-L109.

The basic idea here is that revoked serials from CRLs might correspond to expired certs. So we have to whittle down the revoked list to the "known and revoked" certs. That's why we're iterating over the known list to figure out which serials from the revoked set to insert. In `exclude` we just take all of the known certs (with multiplicity).